### PR TITLE
files/en-us/glossary/javascript/index.md | Add Deno & Bun as server r…

### PR DESCRIPTION
**Pull request by m-dressler**
…untimes

### Description

Add Deno & Bun as server runtimes

### Motivation

Those runtimes have reached public mainstream as well.

### CODE REVIEW

File: files/en-us/glossary/javascript/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ page-type: glossary-definition
 
 {{GlossarySidebar}}
 
-JavaScript (or "JS") is a programming language used most often for dynamic client-side scripts on webpages, but it is also often used on the {{Glossary("Server","server")}}-side, using a runtime such as [Node.js](https://nodejs.org/en).
+JavaScript (or "JS") is a programming language used most often for dynamic client-side scripts on webpages, but it is also often used on the {{Glossary("Server","server")}}-side, using a runtime such as [Node.js](https://nodejs.org/en), [Deno](https://deno.com/), and [Bun](https://bun.sh/).
```
Comment (bsmth) [inline files/en-us/glossary/javascript/index.md:9]: There's a Deno glossary entry, btw -> `{{Glossary("Deno")}}`. I notice we don't have one for Bun yet.

Comment (m-dressler) [reply files/en-us/glossary/javascript/index.md:9]: Are you asking to change the URLs from their respective websites to use internal MDN docs or is this for reference to others?

Comment (bsmth) [reply files/en-us/glossary/javascript/index.md:9]: Sorry for not clarifying, usually we'll link internally in Glossary entries, [like the second paragraph](https://github.com/mdn/content/pull/36923/files#diff-a0814684568d70e0a192c67fe8d73498e512247be7077455cf2e7a304c222723R13) - you may do this with the links in your PR but it's only a suggestion.

Comment (m-dressler) [reply files/en-us/glossary/javascript/index.md]: Updated to use glossary links

**Review by bsmth — APPROVED** [review]
LGTM, thanks!

**Pull request merged**